google/cloud/vision/client.py

```diff
@@ -98,12 +98,22 @@ def annotate(self, image, features):
         :rtype: dict
         :returns: List of annotations.
         """
-        img = Image(image, self)
-        request = VisionRequest(img, features)
+        request = VisionRequest(image, features)
 
         data = {'requests': [request.as_dict()]}
         response = self.connection.api_request(method='POST',
                                                path='/images:annotate',
                                                data=data)
 
         return response['responses'][0]
+
+    def image(self, image_source):
+        """Get instance of Image using current client.
+
+        :type image_source: str or bytes
+        :param image_source: Byte stream of an image or a GCS URI.
+
+        :rtype: :class:`google.cloud.vision.image.Image`
+        :returns: Image instance with the current client attached.
+        """
+        return Image(client=self, image_source=image_source)
```

Comment:
I'm still finding the overload of `bytes` and `str` unpleasant:  I think there should be separate parameters (or maybe even separate factories) for the two cases (one with raw image bytes in hand and one with a GCS URI).